packages/dcos-ui-service/buildinfo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "single_source": {
+    "kind": "url",
+    "url": "http://downloads.mesosphere.com/dcos-ui-update-service/latest/dcos-ui-update-service",
```

Comment:
we cant ship this like this, once we merged this PR to dcos and update the file (which happens automatically though our CI), we will break dcos release branch… Or am I missing something? we need some kind of versioning here.

Reply:
@juliangieseke , we have an open jira about versioning - https://jira.mesosphere.com/browse/DCOS-43672 . I didn't know that this will break dcos release branch. Do you have a suggestion on how we might implement versioning here?

Reply:
https://github.com/dcos/dcos-ui-update-service/pull/8 should resolve the issue

Reply:
Cool, thank you.